scripts/in_container/verify_providers.py

```diff
@@ -898,15 +910,17 @@ def run_provider_discovery():
     subprocess.run(["airflow", "providers", "widgets"], check=True)
     console.print("[bright_blue]List all extra links[/]\n")
     subprocess.run(["airflow", "providers", "links"], check=True)
-    if os.environ.get("USE_AIRFLOW_VERSION", None) == "2.1.0":
+
```

Comment:
Actually we can remove all that. We ONLY check one version - 2.3. All the previous checks can be safely removed.

Reply:
I initially don't now how we use this command that's why I parse USE_AIRFLOW_VERSION. I will drop Airflow 2.1 support